app/scripts/modules/core/src/pagerDuty/PageModal.tsx

```diff
@@ -40,8 +40,13 @@ export class PageModal extends React.Component<IPageModalProps, IPageModalState>
   }
 
   private getDefaultState(props: IPageModalProps): IPageModalState {
-    const defaultSubject = get(SETTINGS, 'pagerDuty.defaultSubject', 'Urgent Issue');
-    const defaultDetails = get(SETTINGS, 'pagerDuty.defaultDetails', '');
+    const {
```

Comment:
I prefer that you access via `ReactInjector.$uiRouter.globals.params`

Reply:
👍